package.json

```diff
@@ -30,13 +30,15 @@
     "mobile"
   ],
   "dependencies": {
+    "@types/color": "^3.0.0",
```

Comment:
this should be in the devDependencies section. Also, can you please use exact version.

Reply:
`npm i --save-dev --save-exact @types/color`